modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/navigation.hbs

```diff
@@ -55,6 +55,7 @@
                         </div>
                     {{/if}}
                     {{> categorization-entry .}}
+                    {{#if assetCategoryFilterDetails}}<style>div.body-container{min-height: 1220px;}</style>{{/if}}
```

Comment:
Thank you Dakshika for mentioning this,
This has already refactored with multiple taxonomy feature.

https://github.com/wso2/product-greg/blob/master/modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/partials/navigation.hbs#L89